src/System.Memory/src/System/ThrowHelper.cs

```diff
@@ -151,6 +151,20 @@ private static Exception CreateArgumentValidationException(Array array, int star
             else
                 return CreateArgumentOutOfRangeException(ExceptionArgument.length);
         }
+
+        //
+        // ReadOnlySequence Slice validation Throws coalesced to enable inlining of the Slice
+        //
+        public static void ThrowArgumentValidationException(long start)
+            => throw CreateArgumentValidationException(start);
+
+        private static Exception CreateArgumentValidationException(long start)
```

Comment:
It is only start. Length should be as second param but we do not pass it because if start is correct then length is incorrect anyway.

Reply:
I understand that. My point is, `CreateArgumentValidationException` can create an exception for either start or for length. If someone passes in negative start value, we know start is invalid, and otherwise, length is invalid. So, the argument name, method name, etc should indicate to the caller what is happening here in some way.

Reply:
I added comment should I renamed this method ?

Reply:
It's a private method so it should be OK as is.